recipes/spec2nexus/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "spec2nexus" %}
+{% set version = "2021.2.0rc0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33fea814b07ce18c77d8b9390be54ef8a7a08f9d8ceb39afdd05ac99442a9f3d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - extractSpecScan = spec2nexus.extractSpecScan:main
+    - spec2nexus = spec2nexus.nexus:main
+    - specplot = spec2nexus.specplot:main
+    - specplot_gallery = spec2nexus.specplot_gallery:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - coverage
+    - docopt
+    - h5py
+    - lxml
+    - matplotlib
```

```suggestion
    - matplotlib-base
```
Sorry, I was unclear. We do want the requirements listed here to match those listed in the pip metadata, but if spec2nexus does not ever open a GUI, then we should still use matplotlib-base here. I believe `pip check` will still work with matplotlib-base because the python module is still matplotlib (only the dependencies exclude qt).

The rest of the dependencies I will trust you to trim with your next update. For example, docopt/coverage? Those sound like dev tools.